bcgithook/install.sh

```diff
@@ -52,6 +52,11 @@ bash_ok=no && [ "${BASH_VERSINFO:-0}" -ge $min_bash_version ] && bash_ok=yes
 # - declare some useful functions
 #
 
+# - shellcheck directives 
```

Comment:
what are these comments for? are you expecting shell check to understand these and disable these checks?

Reply:
pls see reply to comment below

Reply:
I've never done it that way, I've always passed as args using `-e`, like:
- https://github.com/koalaman/shellcheck/wiki/Ignore#ignoring-errors-in-one-specific-run
- https://github.com/marketplace/actions/shellcheck#globally-disable-checks

looking at the CI, not sure its worked either. still complains about SC2034

Reply:
Worth trying with "-e" as well.